projects/ngrx.io/content/examples/store-walkthrough/src/app/state/books.actions.ts

```diff
@@ -1,17 +1,11 @@
-import { createAction, props } from '@ngrx/store';
+import { createActionGroup, props } from '@ngrx/store';
 import { Book } from '../book-list/books.model';
 
-export const addBook = createAction(
-  '[Book List] Add Book',
-  props<{ bookId: string }>()
-);
-
-export const removeBook = createAction(
-  '[Book Collection] Remove Book',
-  props<{ bookId: string }>()
-);
-
-export const retrievedBookList = createAction(
-  '[Book List/API] Retrieve Books Success',
-  props<{ books: ReadonlyArray<Book> }>()
-);
+export const BooksActions = createActionGroup({
+  source: 'Books List/Collection',
```

Comment:
should I create 2 different groups, one for `list` and an other for `collection` ?

Reply:
One for `Books` and one for `Books API` would be better to separate between UI actions and API interactions

Reply:
Yea, I'm thinking the same thing 👍
I first thought of 3 groups, but this would be too much for this example and make it needlessly complex. 2 would be a good fit.

Reply:
@brandonroberts @timdeschryver  I separate UI actions and API interactions in two different groups. Let me know if you prefer having groups in 2 different files